exercises/concept/need-for-speed/.docs/instructions.md

```diff
@@ -1,22 +1,32 @@
 # Instructions
 
-In this exercise you'll be organizing races between various types of remote controlled cars. Each car has its own speed and battery drain characteristics.
+In this exercise you'll be organizing races between various types of remote controlled cars.
+Each car has its own speed and battery drain characteristics.
```

Comment:
not sure if there are specific guidelines for text, but it helps to read by splitting into multiple lines. I can revert if prefered :)

Reply:
This is totally fine!